agents/aws-to-gcp/src/main/scala/org/broadinstitute/transporter/transfer/AwsToGcpRunner.scala

```diff
@@ -287,51 +287,52 @@ class AwsToGcpRunner(
     path: String,
     contentType: `Content-Type`,
     expectedMd5: Option[String],
-    data: Chunk[Byte]
+    data: Stream[IO, Byte]
```

Comment:
The logic here now (mostly) matches the implementation in `monster-storage-libs`. We had hit the same multipart upload bug there, then fixed it, but we never backported the fix.

Note to self: Make a ticket to use `monster-storage-libs` in Transporter and delete these implementations.

Reply:
https://broadinstitute.atlassian.net/browse/DSPDC-522

Reply:
https://broadinstitute.atlassian.net/browse/DSPDC-522